attributes_test.go

```diff
@@ -4,23 +4,25 @@ import (
 	"testing"
 
 	pb "github.com/owulveryck/onnx-go/internal/pb-onnx"
+	
+	"gotest.tools/assert"
```

Comment:
The project is already using https://godoc.org/github.com/stretchr/testify/assert
I do not have any opinion about which package is best (I used the previous one because it is the one Gorgonia is using as well). 
But whatever package is chosen, it should be the convention for the whole onnx-go package to avoid unnecessary dependencies. 
What are the pro of this package in regard to `testify`?

Reply:
Actually, there are no pros since it looks like testify has more complete interface, will fix this in the evening

Reply:
I think it would be more correct to submit this PR with testify and create a separate issue to replace it with `gotest.tools`

Reply:
For the record, I created issue #146